site/hxapi/viewamendcancel/view.md

```diff
@@ -156,6 +156,40 @@ Note the `.js` extension.
 
 ---
 
+### UK Products View Booking (for a transfer)
+
+#### Request
+{% codetabs %}
+{% codetab XML %}
+```
+https://api.holidayextras.co.uk/v1/booking/YourBookingRef?ABTANumber=YourABTANumber&Password=YourPassword&Initials=YourInitials&key=YourKey&Email=test@test.com
+```
+{% endcodetab %}
+{% codetab JSON %}
+```
+https://api.holidayextras.co.uk/v1/booking/YourBookingRef.js?ABTANumber=YourABTANumber&Password=YourPassword&Initials=YourInitials&key=YourKey&Email=test@test.com
+```
+Note the `.js` extension.
+{% endcodetab %}
+{% endcodetabs %}
+
+#### Response
+
+{% codetabs %}
+{% codetab XML %}
+```xml
+{% include_relative examples/_view_uk_transfers_booking_response.xml %}
+```
+{% endcodetab %}
+{% codetab JSON %}
+```json
+{% include_relative examples/_view_uk_transfers_booking_response.json %}
```

Comment:
Any idea why this is highlighted red?

Reply:
I'm not sure, it only appears to happen for json files. Other file types; markdown, and xml don't have this issue.

It happens in multiple parts of this repo: [site/hxapi/productlibrary/index.md](https://github.com/holidayextras/partner-api-docs/blob/cdaa7a2679888416fcc8da84ad2845ff62673b22/site/hxapi/productlibrary/index.md?plain=1)